### PR TITLE
Validator: Add '-1.5' as valid value for discrete CNA

### LINKED ISSUE #2461: Validator: data_CNA should allow 1.5 and -1.5 values
For the public study: msk_impact_2017, validator should not throw an error for decimal vale of 1.5 and -1.5
screen shot 2017-05-18 at 8 52 28 pm

### PR DESCRIPTION
**Pull request by sandertan**
# What? Why?
Fix #2461  .

Changes proposed in this pull request:
- Allow -1.5 ~~and 1.5~~ in the validator.
- Add to unit tests.

Used in MSK IMPACT study. -1.5 is displayed as "Deep Deletion" (same as -2).

### CODE REVIEW

File: core/src/test/scripts/test_data/data_cna_genecol_presence_both.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ ATAD3B	83858	0	-1	0	-1	2
 ATAD3C	219293	0	-1	0	-1	2
 AURKAIP1	54998	0	-1	0	-1	2
 B3GALT6	126792	0	-1	0	-1	2
+ERCC5	2073	0	-1	0	1.5	2
```
Comment (pieterlukasse) [inline core/src/test/scripts/test_data/data_cna_genecol_presence_both.txt]: you can easily add -1.5 here as well to increase your test coverage from 50% to 100% for this case

**Comment by pieterlukasse** [comment]
Looks good :+1: 

For completeness, I would like to have the part of the code that does this rounding identified in this PR. Just in case we may want to come back to this later.

**Comment by sandertan** [comment]
Good point Pieter. I noticed that actually only -1.5 is allowed, not 1.5!

From `ImportTabDelimData.java`
```
public class ImportTabDelimData {
    public static final String CNA_VALUE_AMPLIFICATION = "2";
    public static final String CNA_VALUE_GAIN = "1";
    public static final String CNA_VALUE_HEMIZYGOUS_DELETION = "-1";
    public static final String CNA_VALUE_HOMOZYGOUS_DELETION = "-2";
    public static final String CNA_VALUE_PARTIAL_DELETION = "-1.5";
    public static final String CNA_VALUE_ZERO = "0";
    private HashSet<Long> importSetOfGenes = new HashSet<Long>();
    private HashSet<Integer> importedGeneticEntitySet = new HashSet<>(); 
```

We might also want to update the frontend to distinguish between -1.5 and -2.

**Comment by sandertan** [comment]
@pieterlukasse Can you rereview?

**Comment by n1zea144** [comment]
Can someone rebase this PR?

**Comment by sandertan** [comment]
@n1zea144 Done!

**Pull request merged**